inst/rmarkdown/templates/xaringan/resources/useR.css

```diff
@@ -0,0 +1,51 @@
+.hljs-github {
+  background-image: url("https://www.r-project.org/conferences/useR.png"), linear-gradient(to bottom, white 85%, rgba(227, 227, 227) 95%) ;
+  background-position: 8% 95% , center;
+  background-size: 85px , 100% 100%, cover ;
+}
+
+.hljs-github h1, .hljs-github h2, .hljs-github h3 {
```

Comment:
```suggestion
.remark-slide-content h1, .remark-slide-content h2, .remark-slide-content h3 {
```

Reply:
This makes sense, thank you so much for your feedback!

Reply:
Great. Then I'll merge the PR in a minute. Thanks!